Makefile

```diff
@@ -16,6 +16,7 @@ SIG=$(PKG).asc
 PREFIX?=/usr/local
 SHARE_DIR=share
 DOC_DIR=$(SHARE_DIR)/doc/$(PKG_NAME)
+ZSH_COMP_DIR=$(DESTDIR)$(PREFIX)/$(SHARE_DIR)/zsh/site-functions
```

Comment:
I actually prefer smaller case here. According to [GNU Make manual](https://www.gnu.org/software/make/manual/html_node/Using-Variables.html):

> It is traditional to use upper case letters in variable names, but we recommend using lower case letters for variable names that serve internal purposes in the makefile, and reserving upper case for parameters that control implicit rules or for parameters that the user should override with command options (see [Overriding Variables](https://www.gnu.org/software/make/manual/html_node/Overriding.html#Overriding)).

I didn’t did it to be consistent. I think we can have a follow-up CL to fix all these?

Reply:
For your convenience I just add a commit to demonstrate the idea.